### PR TITLE
Siddharth

### PR DESCRIPTION
**Pull request by sidarun009**
please review sir, I added one more line from my side which is a 3rd line

### CODE REVIEW

File: Hello.js [hunk 1]
```diff
@@ -1 +1,3 @@
-console.log("hello");
\ No newline at end of file
+console.log("hello");
+console.log("sidhu");
+console.log("sidhu added another line for review");
```
Comment (shaikafroz786) [inline Hello.js:3]: you can add this line but honestly, it looks unnecessary to me

**Review by shaikafroz786 — COMMENTED** [review]
Otherwise,looks fine to me overall

File: Hello.js (hunk 1 above)
Comment (sidarun009) [reply Hello.js:3]: Ok sir, i will remove it but later ,let me try merging

**Review by shaikafroz786 — COMMENTED** [review]
okio

**Pull request merged**